core/kazoo_endpoint/Makefile

```diff
@@ -3,4 +3,9 @@ PROJECT = kazoo_endpoint
 
 all: compile
 
+test: kze-test
+
+kze-test:
+	$(MAKE) compile-test -C $(ROOT)/core/kazoo_number_manager/
```

Comment:
The explicit test dep is what I was going for here. I don't need all of kazoo test-compiled when running kazoo_endpoint tests, so this was the shortest path to having kazoo_endpoint tests run successfully.